package.json

```diff
@@ -9,13 +9,16 @@
     "dom-to-image": "^2.6.0",
     "file-saver": "^2.0.2",
     "html2canvas": "^1.0.0-rc.5",
+    "i18next": "^19.6.3",
+    "i18next-xhr-backend": "^3.2.2",
```

Comment:
i18next-xhr-backend is deprecated - [check out the repo README](https://github.com/i18next/i18next-xhr-backend)